src/styles/GlobalStyles.tsx

```diff
@@ -0,0 +1,25 @@
+import {StyleSheet} from 'react-native';
+
+const GlobalStyles = StyleSheet.create({
+  container: {
+    marginTop: 50,
+  },
+  header: {
+    color: '#337AB7',
```

Comment:
I generally prefer having a single file with all the color declarations as constants, that way it's easier to reuse in various places. I don't really mind this here though since I'm going to be working on theming later ;)